grails-app/controllers/org/pih/warehouse/UrlMappings.groovy

```diff
@@ -942,6 +942,11 @@ class UrlMappings {
             action = [POST: "createRequests"]
         }
 
+        "/api/facilities/$facility/cycle-counts/start/batch" {
```

Comment:
@jmiranda I tried to follow your last suggestion that those params should be also bound to a command, but suddenly they aren't :thinking: they are only accessible via `params`.

Reply:
For what it's worth, it did work for me. I'll try to pull up the PR that I was working on to see if I did anything differently. Off the top of my head, the only difference I can remember is that the UrlMapping I used had braces around facility (I remember because Evan commented on it). I doubt that has any bearing since $facility and ${facility} should be equivalent, but worth a shot I guess.

```
"/api/facilities/${facility}/cycle-counts/start/batch"
```

Reply:
Ok, I just tested this again and the command data binding works for GET requests, but not for POSTs. Interesting and annoying.

Reply:
@jmiranda thanks for testing it out. Yeah it's annoying. Btw - the link with the video doesn't work for me.

Reply:
Sorry had to change the permissions on it. Should be accessible now.